stories/SQForm.stories.tsx

```diff
@@ -478,15 +481,15 @@ export const basicFormWithMultiSelect = () => {
         >
           {MOCK_FRIENDS_OPTIONS}
         </SQFormMultiSelect>
-        <Grid item size={2} style={{alignSelf: 'flex-end'}}>
+        <Grid item style={{alignSelf: 'flex-end'}}>
```

Comment:
Also wanted to mention this.  `size` is not a prop on `Grid` so this was literally doing nothing.  There are more instances of this below.